mobile/src/libs/components/button/styles.ts

```diff
@@ -53,6 +59,19 @@ const styles = StyleSheet.create({
   addButtonLabel: {
     color: AppColor.BLUE_300,
   },
+  buttonModalCancel: {
+    backgroundColor: AppColor.GRAY_400,
+    borderRadius: 8,
+    marginHorizontal: 10,
+    paddingHorizontal: 20,
+  },
+
+  buttonModalDelete: {
+    backgroundColor: AppColor.BLUE_300,
+    borderRadius: 8,
+    marginHorizontal: 10,
+    paddingHorizontal: 20,
+  },
```

Comment:
We can create a common style for both and then add backgroundColor based on the button types